components/calendar/src/cal/ethiopian.rs

```diff
@@ -64,53 +67,9 @@ pub enum EthiopianEraStyle {
 #[derive(Copy, Clone, Debug, Hash, Default, Eq, PartialEq, PartialOrd, Ord)]
 pub struct Ethiopian(pub(crate) bool);
 
-/// The inner date type used for representing [`Date`]s of [`Ethiopian`]. See [`Date`] and [`Ethiopian`] for more details.
-///
-/// The year is stored as Amete Alem year
+#[allow(missing_docs)] // not actually public
```

Comment:
Nit: Let's not add `#[missing_docs]`. I would like this type to be public but have no methods on it.

It might be a bug that this type isn't exported, since it can be accessed by `Coptic::DateInner`.

Reply:
I think a very short line of docs is good here. We don't need more.

Yes, these are mostly assoc types, but people might stumble upon them via search. We should probably improve the docs for that case (the current docs aren't great!), but we shouldn't remove them.

Reply:
> the current docs aren't great!

the current docs are not public, because this is not a public type

Reply:
ah, huh. That's fine then